### PR TITLE
CHE-3740; fix debian scipts java install

### LINKED ISSUE #3740: All bitnami stacks do not work
**Reproduction Steps:**
* Go to the dashboard.
* Try to create a template project from any bitnami stack in the stack library.

**Expected behavior:**
* The project should be created and opened in the IDE.

**Observed behavior:**
* The workspace agent does not start.

**Che version:**       5.1.0-SNAPSHOT
**OS and version:** Ubuntu 14.04
**Docker version:**  1.12

**Additional information:**
* Problem can be reliably reproduced: Yes
* See the attachment:
bitnami_stack

### PR DESCRIPTION
**Pull request by mshaposhnik**
### What does this PR do?
Add a OpenJDK installation fix for some Debian-based images

### What issues does this PR fix or reference?
fixes #3740

### CODE REVIEW

File: agents/che-core-api-agent/src/main/resources/agents/scripts/org.eclipse.che.ws-agent.script.sh [hunk 1]
```diff
@@ -98,6 +98,7 @@ elif echo ${LINUX_TYPE} | grep -qi "debian"; then
 
     if [ ${INSTALL_JDK} = true ]; then
         PACKAGES=${PACKAGES}" openjdk-8-jdk-headless";
+        echo "deb http://ftp.de.debian.org/debian jessie-backports main" | ${SUDO} tee --append /etc/apt/sources.list
```
Comment (benoitf) [inline agents/che-core-api-agent/src/main/resources/agents/scripts/org.eclipse.che.ws-agent.script.sh]: shouldn't we use http://httpredir.debian.org/debian ?

Comment (ghost) [reply agents/che-core-api-agent/src/main/resources/agents/scripts/org.eclipse.che.ws-agent.script.sh]: I used that url in a comment as an example. Yes, we should use this one I think. Or install by adding ppa.

Comment (mshaposhnik) [reply agents/che-core-api-agent/src/main/resources/agents/scripts/org.eclipse.che.ws-agent.script.sh]: Ups. `deb http://ftp.debian.org/debian testing main` is the correct link.

Comment (benoitf) [reply agents/che-core-api-agent/src/main/resources/agents/scripts/org.eclipse.che.ws-agent.script.sh]: httpredir.debian.org/ is selecting the closest mirror from the user.

**Review by benoitf — APPROVED** [review]

**Pull request merged**